minkasi/tools/presets_by_source.py

```diff
@@ -240,104 +240,109 @@ def get_bad_tods(name, ndo=False, odo=False):
-    bad_215_09 = [
-        "Signal_TOD-AGBT18B_215_09-s" + str(ss) + ".fits" for ss in [0]
-    ]  # Maybe 6,7 and 8 too?
-    bad_215_10 = [
-        "Signal_TOD-AGBT18B_215_10-s" + str(ss) + ".fits" for ss in [0]
-    ]  # Maybe 6,7 and 8 too?
-    bad_215_11 = [
-        "Signal_TOD-AGBT18B_215_11-s" + str(ss) + ".fits" for ss in [0]
-    ]  # Maybe 6,7 and 8 too?
-    bad_215_12 = [
-        "Signal_TOD-AGBT18B_215_12-s" + str(ss) + ".fits" for ss in [0]
-    ]  # Maybe 6,7 and 8 too?
-    bad_215_13 = [
-        "Signal_TOD-AGBT18B_215_13-s" + str(ss) + ".fits" for ss in [0]
-    ]  # Maybe 6,7 and 8 too?
-    bad_tod = bad_215_01
-    bad_tod.extend(bad_215_02)
-    bad_tod.extend(bad_215_03)
-    bad_tod.extend(bad_215_04)
-    bad_tod.extend(bad_215_05)
-    bad_tod.extend(bad_215_06)
-    bad_tod.extend(bad_215_07)
-    bad_tod.extend(bad_215_08)
-    bad_tod.extend(bad_215_09)
-    bad_tod.extend(bad_215_10)
-    bad_tod.extend(bad_215_11)
-    bad_tod.extend(bad_215_12)
-    bad_tod.extend(bad_215_13)
-
-    bad_175_01 = [
-        "Signal_TOD-AGBT18A_175_01-s" + str(ss) + ".fits" for ss in [9]
-    ]  # Maybe 6,7 and 8 too?
-    bad_175_03 = [
-        "Signal_TOD-AGBT18A_175_03-s" + str(ss) + ".fits" for ss in [32, 36, 37, 38, 40]
-    ]  # Mostly due to low elev.
-    bad_175_04 = [
-        "Signal_TOD-AGBT18A_175_04-s" + str(ss) + ".fits"
-        for ss in [10, 11, 12, 28, 29, 32, 33, 34, 35, 44, 45, 46, 48, 49]
-    ]
-    bad_175_06 = [
-        "Signal_TOD-AGBT18A_175_06-s" + str(ss) + ".fits"
-        for ss in [38, 39, 40, 47, 48, 49, 50, 74, 75, 76, 77, 79, 80, 82, 83, 84]
-    ]
-    # bad_175_06=['Signal_TOD-AGBT18A_175_06-s'+str(ss)+'.fits' for ss in range(100)]
-    bad_175_07 = [
-        "Signal_TOD-AGBT18A_175_07-s" + str(ss) + ".fits"
-        for ss in [12, 17, 19, 20, 22, 24, 25, 26, 27, 29, 30, 31]
-    ]
-    # bad_tod = bad_175_01   ;bad_tod.extend(bad_175_04);  bad_tod.extend(bad_175_06);  bad_tod.extend(bad_175_07)
-    # bad_tod = bad_175_01   ;bad_tod.extend(bad_175_04);  bad_tod.extend(bad_175_07)
-
-    bad_111_02 = [
-        "Signal_TOD-AGBT18B_111_02-s" + str(ss) + ".fits"
-        for ss in [7, 8, 16, 17, 18, 19, 21, 23, 24, 28, 30, 38, 39, 40]
-    ]
-    bad_111_03 = [
-        "Signal_TOD-AGBT18B_111_03-s" + str(ss) + ".fits" for ss in [9, 10, 31]
-    ]
-    bad_111_04 = [
-        "Signal_TOD-AGBT18B_111_04-s" + str(ss) + ".fits"
-        for ss in [11, 12, 13, 14, 18, 19, 20, 21]
-    ]
-    bad_111_05 = ["Signal_TOD-AGBT18B_111_05-s" + str(ss) + ".fits" for ss in [14, 15]]
-    # bad_tod = bad_111_02  ;bad_tod.extend(bad_111_03);  bad_tod.extend(bad_111_04);  bad_tod.extend(bad_111_05)
-
-    bad_092_07 = ["Signal_TOD-AGBT19A_092_07-s" + str(ss) + ".fits" for ss in [25]]
-    bad_tod.extend(bad_175_01)
-    bad_tod.extend(bad_175_04)
-    bad_tod.extend(bad_175_06)
-    bad_tod.extend(bad_175_07)
-
-    if name == "moo1046" or name == "mooj1046":
+    if name == "AGBT18B_215":
```

Comment:
None of this code is going to work correctly. `name`  is the name of the cluster, not the observing session, so this will never evaluate to True. It's also still unclear to me what "conflict" you're concerned about.

Reply:
I had added this functionality as I am concerned that as we build up a catalog of sources there may be scans that are mentioned in the lines in this function that are not in an if statement that may be in conflict with future sources we add in "if" statements. I spoke with Jack and legacy code will not run if we change this functionality now. We decided to resolve this by having me remove the block of concern for MOO 1142 which is AGBT18B_215_04 and revert the rest of the code back to how it was. I will then add in the bad/maybe TODs for just MOO 1142.